src/networkremote/tcpserver.h

```diff
@@ -0,0 +1,30 @@
+#ifndef TCPSERVER_H
```

Comment:
Header guard and file name should match the class name

Reply:
Done